deployments.py

```diff
@@ -353,3 +354,14 @@ def _version() -> str:
             AwsEnv.labs: "0 0 * * *",  # Every day at midnight
         },
     )
+
+    # Vibe Check
+
+    create_deployment(
+        flow=vibe_check_inference,  # pyright: ignore[reportArgumentType]
+        description="Run vibe check inference on a set of concepts and push the results to s3. Optionally provide a custom list of Wikibase IDs to process.",
+        env_schedules={
+            # Once a week, at midday on Sunday
+            AwsEnv.labs: "0 12 * * SUN",
+        },
```

Comment:
was this always weekly before? I thought it could've been daily

Reply:
could definitely be daily! tbh i'm just adding _something_ here, and not paying close attention to the specifics. I expect the policy team will tell us what the correct cadence is when they start using the new version of the tool!